tool/flutter_ci_script_stable.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 declare -ar PROJECT_NAMES=(
+    "add_to_app/android_view/flutter_module_using_plugin" \
```

Comment:
Looks like the android part was never submitted to CI.  We should do that in another PR in case it's hairy and not conflate issues.